db-engine/src/main/java/com/splicemachine/db/iapi/sql/conn/SQLSessionContext.java

```diff
@@ -97,7 +97,7 @@
     /**
      * Get the Group user of this SQL connection context
```

Comment:
nit: work on the grammar. "get the group user of" seems missing out something between `group` and `user`.